testplan/runnable/interactive/base.py

```diff
@@ -797,16 +797,29 @@ def reload_report(self):
                     len(new_report[mt.uid][st.uid].entries)
                 ):  # testcase level
                     try:
-                        # Update new report testcase state
-                        new_report[mt.uid][st.uid].entries[
-                            new_case_index
-                        ] = st[
-                            new_report[mt.uid][st.uid]
-                            .entries[new_case_index]
-                            .uid
-                        ]
+                        tc = new_report[mt.uid][st.uid].entries[new_case_index]
```

Comment:
consider to introduce temporary variables instead of those long indexing chains. Speaking names could make this more(euphemism) readable.

Reply:
Moved most of it to enumerate.